src/main/java/nextstep/ladder/GameUser.java

```diff
@@ -0,0 +1,31 @@
+package nextstep.ladder;
+
+import nextstep.ladder.domain.Line;
+
+public class GameUser {
+	private String userName;
+	private Position position;
+
+	public GameUser(String userName, Position position) {
+		this.userName = userName;
+		this.position = position;
+	}
+
+	public void move(Line line) {
+		if (line.rightIsMovable(this.position)) {
+			this.position.rightMove();
+		} else if (line.leftIsMovable(this.position)) {
+			this.position.leftMove();
+		}
```

Comment:
```suggestion
    this.position = line.move(position);
```
위와 같이 line 상태에 따라 position을 이동하도록 구현하면 어떨까?
즉, line에 메시지를 보낸다.